OpenRA.Mods.Common/UpdateRules/Rules/20200503/RenameCircleOutline.cs

```diff
@@ -0,0 +1,38 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class RenameCircleOutline : UpdateRule
```

Comment:
Was introducing a third term for the border here a silent protest, or unintentional oversight?

Can we please call this either `RenameCircleContrast` or `RenameCircleBorder` to match what it does? Or I guess renaming both `Contrast` and `Border` to `Outline` in the traits would also work 😓

Reply:
That wasn't silent protest. Don't really care. To me all those are actually synonyms, I am just renaming things to please you.